build/yaml/deployBotResources/deployBotResources.yml

```diff
@@ -320,21 +322,10 @@ stages:
             registry: ${{ parameters.dependenciesRegistryDotNetSkills }}
             version: ${{ parameters.dependenciesVersionDotNetSkills }}
 
-# DotNet Composer
-  - template: dotnet/deployComposer.yml
-    parameters:
-      appInsight: "$(INTERNALAPPINSIGHTSNAME)"
-      appServicePlan: "$(INTERNALAPPSERVICEPLANDOTNETNAME)"
-      appServicePlanRG: "$(INTERNALAPPSERVICEPLANWINDOWSRESOURCEGROUP)"
-      azureSubscription: "$(AZURESUBSCRIPTION)"
-      botPricingTier: $env:BOTPRICINGTIER
-      keyVault: "$(INTERNALKEYVAULTNAME)"
-      resourceGroup: "$(INTERNALRESOURCEGROUPNAME)-DotNet"
-      resourceSuffix: $(INTERNALRESOURCESUFFIX)
-      bots:
+        # DotNet Composer bots
         - name: "bffnsimplehostbotcomposerdotnet"
           dependsOn: "Deploy_bffnechoskillbotdotnet"
-          type: "Host"
+          type: "Composer"
```

Comment:
I think now this is fine, but what happens if we need to have a condition that only works for Host + Composer?, maybe a isComposer flag?

Reply:
I take note here because its related and cannot comment on an unchanged line.

What happens to these two tasks?
The DirectLine one will not be executed because the type now is Composer, and the oAuth doesn't exists in Composer, but is just to keep in mind for the isComposer flag change
![image](https://user-images.githubusercontent.com/62260472/120856164-cb61ac00-c555-11eb-84eb-5024477ca7d7.png)

Reply:
I thought about that but didn't seem necessary. In the future, if we want to distinguish between host and skill we can change it to Host Composer and Skill Composer, similar to Skill V3.